templates/var/lib/ansible/playbooks/registry.yml

```diff
@@ -17,7 +17,7 @@
 
   - name: Setup RDO repositories
     yum: name=https://rdoproject.org/repos/rdo-release.rpm state=present
-    when: deployment_type == 'origin' and rpm_check.rc != 0
+    when: deployment_type == 'origin' and rpm_check.rc != 0 and ansible_distribution != 'Red Hat Enterprise Linux'
```

Comment:
Thanks. I updated the PR

On Thu, Jun 23, 2016 at 8:40 PM, Judd Maltin <notifications@github.com>
wrote:

> In templates/var/lib/ansible/playbooks/registry.yml
> <https://github.com/redhat-openstack/openshift-on-openstack/pull/157#discussion_r68291523>
> :
>
> > @@ -17,7 +17,7 @@
> >
> >    - name: Setup RDO repositories
> >      yum: name=https://rdoproject.org/repos/rdo-release.rpm state=present
> > -    when: deployment_type == 'origin' and rpm_check.rc != 0
> > +    when: deployment_type == 'origin' and rpm_check.rc != 0 and ansible_distribution != 'Red Hat Enterprise Linux'
>
> I think I see the problem here:
>
> [root@flannel2-infra ~]# ansible  localhost -m setup | grep ansible_distribution
>         "ansible_distribution": "RedHat",
>         "ansible_distribution_major_version": "7",
>         "ansible_distribution_release": "Maipo",
>         "ansible_distribution_version": "7.2",
> [root@flannel2-infra ~]#
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/redhat-openstack/openshift-on-openstack/pull/157/files/1da4e96c9b12f0f7e5d6bc3509c15579db1e5a9f#r68291523>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AABtZVUPxIffIYZlZlN3nEB8AO77IUqFks5qOtM5gaJpZM4I7fe3>
> .
>